test/queryable/parse-native-query-error.js

```diff
@@ -78,7 +78,7 @@ describe('Queryable ::', function() {
         assert.equal(err.exit, 'queryFailed');
 
         Pack.parseNativeQueryError({
-          nativeQueryError: err.output.error
+          nativeQueryError: err.raw.error
```

Comment:
@mk2 would you mind sending this as a separate PR?

Reply:
@mikermcneil this was fixed in latest machinepack-mysql master branch, so I was pulled it and eliminated from this PR.